airflow/utils/log/logging_mixin.py

```diff
@@ -32,7 +32,7 @@
 
 # Private: A sentinel objects
 class SetContextPropagate(enum.Enum):
-    """:meta private:"""
+    """:meta private:."""
```

Comment:
We should not do this. Is it possible to convince Pydocstyle to skip this?

Reply:
I tried:

```python
class DefaultPolicy:  # noqa: D400
    """:meta private:"""
```

```python
# noqa: D400
class DefaultPolicy:
    """:meta private:"""
```

```python
class DefaultPolicy:
    # noqa: D400
    """:meta private:"""
```

None of them work. This is weird because there is a precise example of what we are trying to achieve in [pydocstyle documentation](http://www.pydocstyle.org/en/2.1.1/usage.html#in-file-configuration). Question of version maybe?

Reply:
Maybe it’s easiest to just give those a proper docstring.

Reply:
Since you did not allow maintainers to push to your branch, I opened a pull request against your repository.

Reply:
Thanks! I'll cherry-pick these changes in this PR